pubspec.yaml

```diff
@@ -20,7 +20,10 @@ dependencies:
   flutter_local_notifications: ^9.2.0
   flutter_polyline_points: ^1.0.0
   fluttertoast: ^8.0.3
-  geocoder: ^0.2.1
+  geocoder:
```

Comment:
These changes are already present in #87. You may want to keep this PR for the issue itself :)